src/datastore/postgres/schema/v5.ts

```diff
@@ -0,0 +1,28 @@
+import { IDatabase } from "pg-promise";
+
+// tslint:disable-next-line: no-any
+export async function runSchema(db: IDatabase<any>) {
+    // Create schema
+    await db.none(`
+        CREATE TABLE metrics_users (
```

Comment:
Looking at this again, this feels useless if we already have a users table. I'm minded to drop it.